From 6d5b40c6fabb864715ca59620ce339b07dd0deae Mon Sep 17 00:00:00 2001
From: ParasailNumerous <305966967+ParasailNumerous@users.noreply.github.com>
Date: Fri, 18 Sep 2026 04:25:32 +0000
Subject: [PATCH 09/15] perf: minify CSS output

Compress CSS output by passing Compressed parameter to
Dart Sass.
---
 tools/gencss.nim | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tools/gencss.nim b/tools/gencss.nim
index ed3aa3c..723b3ff 100644
--- a/tools/gencss.nim
+++ b/tools/gencss.nim
@@ -2,6 +2,7 @@ import sass
 
 compileFile("src/sass/index.scss",
             outputPath = "public/css/style.css",
+            outputStyle = Compressed,
             includePaths = @["src/sass/include"])
 
 echo "Compiled to public/css/style.css"
-- 
2.56.0

